### PR TITLE
Added optional data-confirm-label and data-cancel-label for JS modal dialog

### PR DESCRIPTION
**Pull request by cH40z-Lord**
The changes made allows one to pass in custom labels for the confirm and cancel button. It is especially useful in an i18n environment.

If these data attributes are not provided an English fallback will be used.

### CODE REVIEW

File: Resources/assets/js/removal_confirm.js [hunk 1]
```diff
@@ -55,9 +55,11 @@ jQuery(function($) {
...
     function renderModalContent ($link)
     {
-        var title   = $link.data("original-title") || $link.attr("title");
-        var url     = $link.attr("href");
-        var message = $link.data("confirm");
+        var title        = $link.data("original-title") || $link.attr("title");
+        var url          = $link.attr("href");
+        var message      = $link.data("confirm");
+        var confirmLabel = $link.data("confirm-label") || "Continue";
+        var cancelLabel  = $link.data("cancel-label") || "Cancel";
```
Comment (apfelbox) [inline Resources/assets/js/removal_confirm.js]: Same here: BC break. Please reuse the previous texts here.

Comment (cH40z-Lord) [inline Resources/assets/js/removal_confirm.js]: I've reverted it again.

**Comment by apfelbox** [comment]
Now you should update the compiled JS file, too. :wink:

**Comment by cH40z-Lord** [comment]
Right, different projects, different workflows ;)

**Pull request merged**